docs/sources/datasources/azuremonitor.md

```diff
@@ -413,3 +414,53 @@ Application Insights and Insights Analytics are two ways to query the same Azure
 > **Note** In Grafana 8.0, Application Insights and Insights Analytics will be deprecated and made read-only in favor of querying this data through Metrics and Logs. Existing queries will continue to work, but you cannot edit them.
 
 To prepare for this upcoming change, Application Insights queries can now be made in Metrics, under the "microsoft.insights/components" Namespace. Insights Analytics queries cannot be made within Logs with KQL at this time.
+
+## Query Azure Resource Graph service
+
+{{< docs-imagebox img="/img/docs/v80/azuremonitor-azure-resource-graph.png" class="docs-image--no-shadow" caption="Azure Resource Graph Editor" >}}
```

Comment:
@joshhunt or @sunker, no images should be in version folders. They should be in folders named after the subect, like azure-monitor. Please review the Grafana style guide for instructions about [Images](https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-style-guide.md#images).

Reply:
Also, please organize this section with subheadings that indicate what each portion is about. Right now, it is hard to follow.

Reply:
Image fixed! Can you please check to see if it looks ok @oddlittlebird?

Reply:
Yep, looks good

Reply:
@shuotli - The image is correct now, but the comment about subheadings still needs to be addressed.